app/components/support_interface/state_explanation_component.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2 class='govuk-heading-l' id='<%= state_name %>'><%= t "application_states.#{state_name}.name" %></h2>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-body">
+      <%= t "application_states.#{state_name}.description" %>
+    </div>
+
+    <% if state.events.any? %>
+      <h4 class='govuk-heading-m'>Actions</h4>
+
+      <ul class="govuk-list">
+      <% state.events.each do |_, events| %>
+        <% event = events.first # this is an array, but will always have a single item %>
+        <li>
+          <strong><%= StateDiagram.event_name(state.name, event) %></strong> (by <%= I18n.t!("events.#{state.name}-#{event}.by") %>). <%= t "events.#{state_name}-#{event.name}.description" %>
+
+          This will transition the application to the <%= govuk_link_to "#{event.transitions_to.to_s.humanize} state", "##{event.transitions_to.to_s}" %>.
+          <br><br>
+          <% if I18n.exists?("events.#{state_name}-#{event.name}.emails") %>
+            Emails sent:
+
+            <ul class='govuk-list govuk-list--bullet'>
+            <% t("events.#{state_name}-#{event.name}.emails").each do |e| %>
+              <li><%= govuk_link_to e.gsub('-', ' '), "/rails/mailers/" + e.gsub('-', '/') %></li>
+            <% end %>
+            </ul>
+          <% else %>
+            No emails are sent with this transition.<br><br>
+          <% end %>
+        </li>
+      <% end %>
+      </ul>
+    <% else %>
+      <div class="govuk-body">
+        This state doesn't have any more actions.
+      </div>
+    <% end %>
+  </div>
+
+  <div class="govuk-grid-column-one-third">
+    <%= image_tag support_interface_process_diagram_path(state: state_name), style: 'float:right; max-width: 100%;' %>
```

Comment:
The `float: right` could be a bit wonky on mobile, but this isn't a high priority. It should still be usable even like this. An improvement would be to use a CSS class and to only float right on desktop using a media query.